src/locale/pt-br.json

```diff
@@ -1,11 +1,12 @@
 {
     "locale"         : "pt-BR",
-    "dateFormat"    : "dd/mm/yyyy",
-    "timeFormat"    : "h24",
-    "currency"       : "^[-+]?[0-9]*[\\.]?[0-9]+\\€$",
-    "currencyPrefix" : "",
-    "currencySuffix" : "€",
-    "groupSymbol"    : ",",
+    "date_format"    : "dd/mm/yyyy",
```

Comment:
These changes seem to be wrong too. Correct variable name is dateFormat and timeFormat, not date_format and time_format. I refactored it some time ago to make same naming convention